app/assets/javascripts/survey_trip_stops.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
```

Comment:
You don't like coffeescript?

Personally I vote that we move towards using it.